projects/core/src/user/store/reducers/index.ts

```diff
@@ -25,17 +25,17 @@ import {
   PRODUCT_INTERESTS,
   REGIONS,
   RegionsState,
+  UserState,
```

Comment:
Aren't there soome reducer related classes/files that should be marked as deprecated?

I know they aree not part of the publiic API, but wee still need this for initernal use I thinik.

Reply:
The thing is: I cannot just add the deprecation doc for a property in an object. `getReducers` is a function returning an object (not an interface). Similar for effect, it is an array of effects. When we do the cleanup (clean the user state), the reducer and effect will have errors if they are not be cleaned.

Reply:
It should be possible to flag each exported const, function, class with a deprecated comment for each corresponding file that was moved to the cart lib.  These files may be deleted months from now and a deprecated comment would go a long way to allow whoever performs the task to be confident about what he is about to remove (and often the tickets ask specifically to remove @depreecated code only)

This it what it looks like for reducers and effects moved to cart lib.  It seems to work ok.:

<img width="913" alt="Screen Shot 2021-07-21 at 5 11 25 PM" src="https://user-images.githubusercontent.com/7065346/126561495-5616c44e-d072-4202-894b-945121c1233b.png">
<img width="924" alt="Screen Shot 2021-07-21 at 5 10 54 PM" src="https://user-images.githubusercontent.com/7065346/126561497-c9a78d26-07a4-4d0b-9bbc-d0d1be48ffde.png">